src/generator/AutoRest.Go/Extensions.cs

```diff
@@ -904,9 +904,9 @@ public static bool IsBodyParameter(this IParameter p)
         /// <returns></returns>
         public static string GetConstraint(string name, string constraintName, string constraintValue, bool chain = false)
         {
-            var value = constraintName == Constraint.Pattern.ToString()
-                                             ? string.Format("\"{0}\"", constraintValue.Replace("\\", "\\\\"))
-                                             : constraintValue;
+            var value = constraintName == Constraint.Pattern.ToString()  
```

Comment:
I checked constraint value won't always come as string.  It will be...
`Integer for Min-Max length constraint`
`String for pattern constraint`
`and so forth..`

No change here. Thanks for pointing though. Good observation.